src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -464,11 +464,25 @@ private static TextSpan GetActiveSpan(SwitchExpressionSyntax node, SwitchExpress
         protected override bool AreEquivalent(SyntaxNode left, SyntaxNode right)
             => SyntaxFactory.AreEquivalent(left, right);
 
-        private bool AreEquivalentModifiersIgnoringUnsafe(SyntaxTokenList oldModifiers, SyntaxTokenList newModifiers)
+        private bool AreEquivalentModifiersIgnoringGivenModifier(SyntaxTokenList oldModifiers, SyntaxTokenList newModifiers, params SyntaxKind[] ignoredModiferKinds)
```

Comment:
I'd define 
`private readonly static SyntaxKind[] s_ignoredMemberModifiers = { SyntaxKind.UnsafeKeyword, SyntaxKind.Async };`
 and use it as an argument in these calls. That would both avoid extra allocation and make sure we are ignoring the same modifiers in all members. We can ignore `async` in all members since it is valid on methods and won't be actually present on other members.